### DISCUSSION TITLE
Assertion failure due to incorrectly detected infeasibility

### DISCUSSION THREAD

**Discussion by XTOCON** [ERGO-Code/HiGHS#1008] [Q&A] [answered]
I'm using the Python 10 Scipy 1.9.3 interface to your Highs library, and encounter the following issue;

For apparently random cases, I get the following error message:

Running HiGHS 1.2.0 [date: 2021-07-09, git hash: n/a]
Assertion failed: infeasible_ == 0, file ..\..\scipy\_lib\highs\src/mip/HighsDomain.h, line 483

For these cases I'm 100% sure the problem is feasible.

Could you please explain how I could resolve this issue?

Thank you

Christophe

**Comment by jajhall** [comment, marked as answer]
This is a `SciPy` build error that (for the benefit of others reading this discussion) will be fixed in SciPy 1.10, when it is released in late December or early January
